### PR TITLE
Feature/design update v2

### PR DESCRIPTION
**Pull request by Adamws33**
updated cal view to have search box full width, re adjusted location page text and map, added a grey background to selected crops on cal and list views.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.106.94:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.106.94:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSelector/CropCalendarView/RenderCrops.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const RenderCrops = ({ setModalOpen, modalOpen, setModalData }) => {
         key={`cropRow${index}`}
         style={hasGoalRatingTwoOrLess(selectedGoalsRedux, crop) ? { opacity: '0.3' } : {}}
       >
-        <TableCell sx={{ padding: 0 }}>
+        <TableCell sx={{ padding: 0, backgroundColor: selectedCropIdsRedux.includes(crop.id) && '#EAEAEA' }}>
```
Comment (YazidZidane) [inline src/pages/CropSelector/CropCalendarView/RenderCrops.js:53]: Is it possible to set backgroundColor to the `<TableRow/>` instead setting it in every `<TableCell/>`?

File: src/pages/CropSelector/CropTable/CropTableCard.js [hunk 2]
```diff
@@ -23,7 +23,16 @@ const CropTableCard = ({ crop, indexKey, showGrowthWindow }) => {
     <>
       {selectedGoalsRedux.length > 0
         && selectedGoalsRedux.map((goal, index) => (
-          <TableCell size="small" style={{ textAlign: 'center' }} key={index} className="goalCells">
+
+          <TableCell
+            size="small"
+            style={{
+              textAlign: 'center',
+              backgroundColor: selectedCropIdsRedux.includes(crop.id) && '#EAEAEA',
```
Comment (YazidZidane) [inline src/pages/CropSelector/CropTable/CropTableCard.js:31]: same here, the color has been set in `<TableRow/>` in `RenderTableItems.js`

Comment (Adamws33) [reply src/pages/CropSelector/CropTable/CropTableCard.js]: good catch i fully expected to have errors in doing the whole row so i approached it as a per cell issue.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://135.237.106.94:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**